aws/logs_monitoring/cache.py

```diff
@@ -442,3 +442,36 @@ def get(self, log_group):
             self.tags_by_id[log_group] = log_group_tags
 
         return log_group_tags
+
+
+resource_group_tagging_client = boto3.client("resourcegroupstaggingapi")
+RESOURCES_PER_PAGE = 100
+LRU_CACHE_SIZE = 50
+STEP_FUNCTIONS_TAGS_CACHE_TTL = 3600  # an hour
+
+
+def get_ttl_hash(ttl=STEP_FUNCTIONS_TAGS_CACHE_TTL) -> float:
+    """Return the same value within `ttl` seconds"""
```

Comment:
I don't understand how this method works. What is a TTL hash? How does floor dividing the unix epoch by 3600 get you a TTL Hash?

Reply:
Is it just like... the unix hour? Every time there's a new hour, the cache gets invalidated? That makes sense